src/cmakeTaskProvider.ts

```diff
@@ -551,6 +575,83 @@ export class CustomBuildTaskTerminal implements vscode.Pseudoterminal, proc.Outp
         }
     }
 
+    private async runPackageTask(): Promise<any> {
+        this.writeEmitter.fire(localize("package.started", "Package task started...") + endOfLine);
+
+        const project: CMakeProject | undefined = this.getActiveProject();
+        if (!project || !await this.isTaskCompatibleWithPresets(project)) {
+            return;
+        }
+        telemetry.logEvent("task", {taskType: "package", useCMakePresets: String(project.useCMakePresets)});
+        await this.correctTargets(project, CommandType.package);
+        const cmakeDriver: CMakeDriver | undefined = (await project?.getCMakeDriverInstance()) || undefined;
+
+        if (cmakeDriver) {
+            let packagePreset: preset.PackagePreset | undefined;
+            this.preset = await this.resolvePresetName(this.preset, project.useCMakePresets, CommandType.package);
+            if (this.preset) {
+                packagePreset = await project?.expandPackagePresetbyName(this.preset);
+                if (!packagePreset) {
+                    log.debug(localize("package.preset.not.found", 'Package preset not found.'));
+                    this.writeEmitter.fire(localize("cpack.failed", "Package preset {0} not found. CPack failed.", this.preset) + endOfLine);
+                    this.closeEmitter.fire(-1);
+                    return;
+                }
+            }
+            const result: number | undefined = cmakeDriver ? await project?.runCPack(cmakeDriver, packagePreset, this) : undefined; //.runCPackCustomized(cmakeDriver, packagePreset, this) : undefined;
```

Comment:
What does the commented part of this line do?

Reply:
Artefact of an older messier state of the feature implementation. Removed.